### ISSUE TITLE
Option to show what's currently available

### ISSUE THREAD

**Issue by hacshacdgacs** [Fallenbagel/jellyseerr#658]
### Description

I use Jellyseer to help clean up my library so having a filter that allows me to see only what's currently on my server would be useful. Currently I'm having to clean up through radarr/sonarr which then confuses Jellyseer on what's actually available or not.

### Desired Behavior

A filter that allows you to see what's already available on your media server. 

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by 8321876218** [comment]
Is this something different than "Hide Available Media (Experimental)" in Settings->General?

**Comment by hacshacdgacs** [comment]
Yes, it's the exact opposite in that it'd show only the available media

**Comment by hacshacdgacs** [comment]
> Is this something different than "Hide Available Media (Experimental)" in Settings->General?
> 
> Edit: I re-read your question and I think I see what you're asking now. You'd like to exact opposite; to view ONLY media that is currently available in your library.
> 
> Jellyseerr should run periodic scans shown in Settings->Jobs & Cache that sync with Jellyfin/Radarr/Sonarr. Those scans can be ran manually if needed. Does that not suite your needs?
> 
> If you'd like to describe your setup, I might have some input on alternative media management methods as well.

I have jellyseer synced to radarr and sonarr with those scans running. If I ever delete something through sonarr or radarr jellyseer doesn't pick up on it and still marks it as available on my emby server. When I delete through jellyseer however it works all the way through like expected. Usually I try and delete through jellyseer but when doing a mass cleanup it's hard when you can't only see what's available.

**Comment by Fallenbagel** [comment]
That should be fixed once the #524 is merged

**Comment by wacomoto** [comment]
The two way sync is great but I also use Jellyseer to delete media when I'm not at my computer. Being able to view all available would be a very useful media management feature.

**Comment by Jeeere** [comment]
I would also like to see this implemented

**Comment by meszolym** [comment]
This would still be great. I would say this could be a filter in the movies and series sections, something like "availability status", and then we could filter based on that.

**Comment by ryanmfransen** [comment]
Hi - I'm currently working on an implementation for this. I have a working prototype locally and am fine-tuning things right now. I plan to submit a PR soon!

Current implementation direction:

Added a new built-in Discover row: My Media Library, based on media availbility
Row is local-library based (not TMDB discover filtering), supports toggle/reorder in Discover customization
Added a dedicated See More page at /discover/library
I’m currently validating and polishing this implementation for PR readiness (behaviour, i18n, and maintainer feedback items)
